R/utils-read.R

```diff
@@ -9,6 +9,31 @@ sizeOf <- function(df){
   return(nrow(df))
 }
 
+#' Returns whether or not all required columns are present
+#' 
+#' 
+emptyData <- function(data, name, requiredFields){
```

Comment:
This seems like it belongs in utils-validation. The function name also doesn't  really describe what's happening.